src/pages/Home.jsx

```diff
@@ -18,19 +18,22 @@ export const Home = (props) => {
 
   return (
     <div className="home">
-      <h1>Welcome to your Smart Shopping List!</h1>
-      {token ? (
-        <button>
-          <Link to="/list">View your shopping list</Link>
-        </button>
-      ) : (
-        <>
+      <header className="hero">
+        <img src={HeroImage} alt="Logo" />
+      </header>
+      <main>
+        <h1>Welcome to your Smart Shopping List!</h1>
+        {token ? (
+          <OrangeButton>
+            <Link to="/list">View your shopping list</Link>
+          </OrangeButton>
+        ) : (
           <OrangeButton onClick={() => generateToken()}>
-            Create new List
+            Create a new list
```

Comment:
Thanks for updating this. I was working off exactly what was on the wireframe. I should have used a more holistic approach.